MainApp/MainApp.xcodeproj/project.pbxproj

```diff
@@ -313,14 +343,28 @@
 		923249A224C03FEF00FDFC3E /* Profile */ = {
 			isa = PBXGroup;
 			children = (
+				1EBF013024D0A7690098C24A /* ProfileLandingCoordinator.swift */,
+				1EBF013324D0A7690098C24A /* ProfileLandingCoordinatorProtocol.swift */,
+				1E44DC5B24D856B400025AE8 /* EditProfileCoordinator.swift */,
+				1E44DC5D24D856C000025AE8 /* EditProfileCoordinatorProtocol.swift */,
 			);
 			path = Profile;
 			sourceTree = "<group>";
 		};
 		923249A324C03FF600FDFC3E /* Profile */ = {
 			isa = PBXGroup;
 			children = (
+				1EBF012324D0A6130098C24A /* ProfileLandingController.xib */,
+				1EBF012924D0A65F0098C24A /* ProfileLandingController.swift */,
+				1EBF012B24D0A6A00098C24A /* ProfileLandingViewModel.swift */,
+				1EBF012D24D0A6B00098C24A /* ProfileLandingProtocol.swift */,
+				1EBF014924D1E5450098C24A /* CustomTableViewCell.swift */,
+				1EBF014B24D1E5510098C24A /* CustomTableViewCell.xib */,
 				923249AB24C0406B00FDFC3E /* Resources */,
+				1E44DC5124D855C600025AE8 /* EditProfileController.swift */,
+				1E44DC5524D8560400025AE8 /* EditProfileController.xib */,
+				1E44DC5724D8562D00025AE8 /* EditProfileProtocol.swift */,
```

Comment:
please create group inside feature/profile called edit and put the edit related stuff inside

Reply:
Moved to its own folder